stock_picking_delivery_info_computation/models/stock_picking.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.depends(
+        'product_id',
```

Comment:
This field doesn't exist at picking level

Reply:
https://github.com/odoo/odoo/blob/10.0/addons/stock/models/stock_picking.py#L289

Perhaps is not relevant on this computation but the field exist and I keep on the definition as it comes from the standard code: https://github.com/odoo/odoo/blob/10.0/addons/delivery/models/stock_picking.py#L105

But as your following comment https://github.com/OCA/delivery-carrier/pull/208#discussion_r307431468 I don't need to repeat this definition, so I will remove this part

Reply:
Uhm, a related to a one2many field... Not sure how that behaves, but OK, we don't need to repeat it.